plugins/module_utils/data_set.py

```diff
@@ -2102,6 +2102,59 @@ def _build_volume_string_iehprogm(volumes):
                 volume_string += single_volume_string + ")\n"
         return volume_string
 
+    @staticmethod
+    def check_if_data_set_migrated(name):
+        """Compares the output of datasets.list_dataset_names with and
+        without migrated data set names to check if name has been migrated.
+
+        Parameters
+        ----------
+        name : str
+            Name of a data set.
+
+        Returns
+        -------
+        bool
+            Whether the data set has been migrated.
+        """
+        has_been_migrated = False
+
+        non_migrated_list = datasets.list_dataset_names(name, migrated=False)
+        migrated_list = datasets.list_dataset_names(name, migrated=True)
+
+        if name in migrated_list and name not in non_migrated_list:
```

Comment:
This comparison will not work if 'name' contains wild cards.  Check if you can see if wildcards could possibly get here, and consider fnmatch functions If needed.  https://docs.python.org/3/library/fnmatch.html

Reply:
At this point variable `name` is not a wildcard, inside the data set handler there is only one data set name.

Reply:
The module only accepts querying a single resource. If there's a `*` or `?` present in `name`, the arg parser already rejects it. Adding wording to the documentation to emphasize this.